pages/fundamentals/telemetry.mdx

```diff
@@ -23,19 +23,33 @@ While a Memgraph database instance is running and an open internet connection is
 available, the following data will be sent to and stored on Memgraph's servers:
 
 - **Information about the host machine**
-    - CPU model
-    - Memory information
-    - Host OS
-    - Kernel information
-    
+  - CPU model
+  - Memory information
+  - Host OS
+  - Kernel information
+<br/>    
+
+- **Information about the clients**
+  - client name
+  - supported and agreed Bolt version
+  - number of sessions from each clients
+  - number of executed queries
+  - authentication type (user or anonymous)
+
+- **Information about active databases**
+  - disk storage and usage
+  - RAM usage
+  - if durability and SSL is enabled
```

Comment:
SSL is on the Memgraph instance level (not under each database config)? @MarkoBarisic please verify 🙏

Reply:
yes SSL should be on Memgraph instance level

Reply:
so leave it to be just `if durabilty and SSL is enabled`